app/app.go

```diff
@@ -143,13 +153,12 @@ var (
 		minttypes.ModuleName:           {authtypes.Minter},
 		stakingtypes.BondedPoolName:    {authtypes.Burner, authtypes.Staking},
 		stakingtypes.NotBondedPoolName: {authtypes.Burner, authtypes.Staking},
-		ibctransfertypes.ModuleName:    {authtypes.Minter, authtypes.Burner},
+		// ibctransfertypes.ModuleName:    {authtypes.Minter, authtypes.Burner},
 		// this line is used by starport scaffolding # stargate/app/maccPerms
```

Comment:
Can delete this comment?

Reply:
yes! removed all of them

https://github.com/celestiaorg/celestia-app/pull/399/commits/f201a83a9ee61c97d5216b4132ada95113813265